INSTALL.md

```diff
@@ -17,7 +17,7 @@ for installing and getting started with the balena CLI on Windows!
 3. Add the `balena-cli` folder to the system's `PATH` environment variable. See instructions for:
    [Windows](https://www.computerhope.com/issues/ch000549.htm) |
    [Linux](https://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux-unix) |
-   [macOS](https://stackoverflow.com/questions/22465332/setting-path-environment-variable-in-osx-permanently).
+   [macOS](https://www.architectryan.com/2012/10/02/add-to-the-path-on-mac-os-x-mountain-lion/#.Uydjga1dXDg)
```

Comment:
Yeah, I think this is an improvement over the previous URL. Developers or some advanced users might prefer to modify the `~/.profile` or `~/.bash_profile` or `~/.bashrc` files, but chances are that those users would not need to refer to those instructions anyway...